magda-sleuther-format/src/onRecordFound.ts

```diff
@@ -19,9 +19,10 @@ import MeasureEvalResult from "./format-engine/MeasureEvalResult";
 let synonymObject = require("./format-engine/synonyms.json");
 
 //import { Snapshot } from "../../magda-typescript-common/src/format/MeasureSnapShot";
-let mochaObject = {
+export let mochaObject = {
```

Comment:
If you use `nock` to mock the return to the registry instead of calling `recordFormatAspectTest` then you can get rid of this completely :).